fbfmaproom/fbfmaproom.py

```diff
@@ -1137,6 +1166,9 @@ def download_table():
     df["time"] = time
     df["bad_year"] = df["bad_year"].astype("float") # to acommodate NaN as missing value indicator
 
+    # rename the obs column "obs" for backwards compatibility
+    df = df.rename(columns={obs_dataset_key: "obs"})
```

Comment:
backwards compatibility with what?

Reply:
Existing clients of this HTML API, namely Max's R Shiny app.